src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -534,16 +534,16 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
                     .AddHeader(Alignment.Right, label: "NPM(K)", width: 7)
-                    .AddHeader(Alignment.Right, label: "PM(K)", width: 8)
-                    .AddHeader(Alignment.Right, label: "WS(K)", width: 10)
+                    .AddHeader(Alignment.Right, label: "PM(M)", width: 8)
+                    .AddHeader(Alignment.Right, label: "WS(M)", width: 10)
                     .AddHeader(Alignment.Right, label: "CPU(s)", width: 10)
                     .AddHeader(Alignment.Right, width: 6)
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
                         .AddScriptBlockColumn("[long]($_.NPM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.PM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.WS / 1024)")
+                        .AddScriptBlockColumn("\"{0:N3}\" -f [float]($_.PM / 1MB)")
```

Comment:
I think we should use `N2`.

A process using 5kb will show `.01` because that gets rounded up, and very few processes use less than 5kb.

I also noticed in your screenshot that the CPU is showing 3 places after the decimal on Linux but we only see 2 on Windows.  I think it's reasonable to use `N2` for CPU to be consistent b/w Windows and Linux.